File: Characters/Stats.py
class Stats:
    def __init__(self):
        self.health = 100
        self.energy = 100
        self.hunger = 0
        self.thirst = 0
        self.strength = 5
        self.endurance = 5
        self.speed = 5
        self.perception = 5
        self.intelligence = 5
        self.knowledge = 5
        self.experience = 0
        self.will = 5
        self.patience = 5
        self.flexibility = 5
        self.balance = 5
        self.charisma = 5

traits = {
    'physical': {'strength': 0, 'speed': 0, 'aggression': 0, 'health': 0, 'appeal': 0},
    'social': {'humility': 0, 'temperament': 0, 'generosity': 0, 'loyalty': 0, 'honesty': 0},
    'Emotional': {'Spontaneity': 0, 'Mannerism': 0, 'Rage': 0},
    'Behavioral': {'Curiosity': 0, 'Dependency': 0, 'Confidence': 0, 'Ambition': 0},
    'Intellectual': {'Creativity': 0, 'Concentration': 0, 'Intelligence': 0}
}

occupation = {
    'Royalty': ["King", "Queen", "Lord", "Lady"],
    'Military': ["Guard", "Soldier", "Captain", "Commander", "General"],
    'Civilian': ["Bard", "Alchemist", "Tanner", "Healer", "Farmer", "Blacksmith", "Barkeep", "Barmaid", "ShopKeep", "Apprentice", "Assistant", "Fletcher", "Butcher", "Weaver", "Potter"],
    'Labour': ["Miner", "Hunter", "Athlete", "Scavenger", "Servant"],
    'Criminal': ["Thief", "Murderer", "Assassin", "Pirate", "Bandit"],
    'Traversal': ["Slave", "Settler", "Adventurer", "Explorer"],
    'Dangerous': ["Mercenary"],
    'School': ["Doctor", "Teacher", "Scholar", "Genius"],
    'Specialist': ["Researcher", "Engineer", "Inventor", "Architect", "Plumber", "Pilot"],
    'Other': ["Idiot", "Jester", "Judge", "Executioner", "Vagrant", "Priest"]
}

my_character = {
    'stats': Stats(),
    'traits': traits,
    'occupation': occupation
}

print(my_character)

File: Characters/Stats.cs
using System;
using System.Collections.Generic;

public class Stats
{
    public int health = 100;
    public int energy = 100;
    public int hunger = 0;
    public int thirst = 0;
    public int strength = 5;
    public int endurance = 5;
    public int speed = 5;
    public int perception = 5;
    public int intelligence = 5;
    public int knowledge = 5;
    public int experience = 0;
    public int will = 5;
    public int patience = 5;
    public int flexibility = 5;
    public int balance = 5;
    public int charisma = 5;
}

class Program
{
    static void Main(string[] args)
    {
        Dictionary<string, Dictionary<string, int>> traits = new Dictionary<string, Dictionary<string, int>>
        {
            { "physical", new Dictionary<string, int> { { "strength", 0 }, { "speed", 0 }, { "aggression", 0 }, { "health", 0 }, { "appeal", 0 } } },
            { "social", new Dictionary<string, int> { { "humility", 0 }, { "temperament", 0 }, { "generosity", 0 }, { "loyalty", 0 }, { "honesty", 0 } } },
            { "Emotional", new Dictionary<string, int> { { "Spontaneity", 0 }, { "Mannerism", 0 }, { "Rage", 0 } } },
            { "Behavioral", new Dictionary<string, int> { { "Curiosity", 0 }, { "Dependency", 0 }, { "Confidence", 0 }, { "Ambition", 0 } } },
            { "Intellectual", new Dictionary<string, int> { { "Creativity", 0 }, { "Concentration", 0 }, { "Intelligence", 0 } } }
        };

        Dictionary<string, List<string>> occupation = new Dictionary<string, List<string>>
        {
            { "Royalty", new List<string> { "King", "Queen", "Lord", "Lady" } },
            { "Military", new List<string> { "Guard", "Soldier", "Captain", "Commander", "General" } },
            { "Civilian", new List<string> { "Bard", "Alchemist", "Tanner", "Healer", "Farmer", "Blacksmith", "Barkeep", "Barmaid", "ShopKeep", "Apprentice", "Assistant", "Fletcher", "Butcher", "Weaver", "Potter" } },
            { "Labour", new List<string> { "Miner", "Hunter", "Athlete", "Scavenger", "Servant" } },
            { "Criminal", new List<string> { "Thief", "Murderer", "Assassin", "Pirate", "Bandit" } },
            { "Traversal", new List<string> { "Slave", "Settler", "Adventurer", "Explorer" } },
            { "Dangerous", new List<string> { "Mercenary" } },
            { "School", new List<string> { "Doctor", "Teacher", "Scholar", "Genius" } },
            { "Specialist", new List<string> { "Researcher", "Engineer", "Inventor", "Architect", "Plumber", "Pilot" } },
            { "Other", new List<string> { "Idiot", "Jester", "Judge", "Executioner", "Vagrant", "Priest" } }
        };

        Dictionary<string, object> myCharacter = new Dictionary<string, object>
        {
            { "stats", new Stats() },
            { "traits", traits },
            { "occupation", occupation }
        };

        foreach (var kvp in myCharacter)
        {
            Console.WriteLine($"{kvp.Key}: {kvp.Value}");
        }
    }
}
